test/exe/Main.hs

```diff
@@ -1121,7 +1121,7 @@ suggestImportTests = testGroup "suggest import actions"
       void (skipManyTill anyMessage message :: Session WorkDoneProgressEndNotification)
       _diags <- waitForDiagnostics
       -- there isn't a good way to wait until the whole project is checked atm
-      when waitForCheckProject $ liftIO $ sleep 0.5
+      when waitForCheckProject $ liftIO $ sleep 1
```

Comment:
Times in tests make me very nervous. Shouldn't you just wait for the diagnostics to get published?

Reply:
these tests are checking for import suggestions, which rely either on the full project being checked (for local imports) or the transitive build-depends being loaded (for package imports). Both actions happen asynchronously and ghcide does not send any notification that can be used to wait currently. Would be easy to fix but certainly doesn't belong in this PR